### DISCUSSION TITLE
PedalinoMini Case1 link unavailable for sponsors

### DISCUSSION THREAD

**Discussion by AlexeiUjegov** [alf45tar/PedalinoMini#418] [Q&A]
Hello! Awesome project! Many thanks for contributing your time for making it! I really want to build it!

Unfortunately, link for Case1 (which is available only for sponsors) leads to 404 page. (I am an active sponsor).

Am I right that it should be stl file for 3d-printer?

**Comment by alf45tar** [comment]
Please check again the link. I added you to the private repository.
Yes, they are STL files.

**Reply by upgrader-dev** [reply]
Same issue with the STL files for pedalinoMini8... any idea ?

**Reply by alf45tar** [reply]
Check you email. The invitation for private repository has been sent.

**Comment by upgrader-dev** [comment]
That’s ok thank youBenjamin Ghedini Le 15 juin 2024 à 15:31, alf45tar ***@***.***> a écrit :﻿
Check you email. The invitation for private repository has been sent.
